pkgs/tools/misc/fontforge/default.nix

```diff
@@ -10,6 +10,8 @@
 stdenv.mkDerivation rec {
   name = "fontforge-${version}";
   version = "20160404";
+  versionModtime = "1459728000"; # unix timestamp of ${version}
```

Comment:
Oh, this is my question which I stuck weekly but never asked yet.
How to execute an external pure function from nix? Only by making a small detivation? It is too bloated a code and works very slow when there are tens of such ```ffi calls```